test/comparison-tests/projectReferencesSymLinks/expectedOutput-4.1/patch0/output.txt

```diff
@@ -6,10 +6,10 @@ asset ../../lib/tsconfig.tsbuildinfo 2.73 KiB [emitted]
 cached modules 408 bytes [cached] 2 modules
 ./src/index.ts 108 bytes [built] [code generated] [1 error]
 
-ERROR in app/src/index.ts
+ERROR in app\src\index.ts
```

Comment:
hmm? why is this a backslash?

btw. webpack usually would display this as `ERROR in ./app/src/index.ts`

Reply:
I suspect this is just ts-loader surfacing up what TypeScript hands over:

https://github.com/TypeStrong/ts-loader/blob/c2d1ca2d5ec6fbca3f4b8c5e1e755e61af719a84/src/utils.ts#L77

So if you're running on a Windows machine it's likely `\` country